components/static-pages/privacypolicy.md

```diff
@@ -23,37 +27,89 @@ When you access or use our Services, we automatically collect information about
 
 We may also obtain information from other sources. For example, if you connect your account with an account you have with a third-party service (such as GitHub), we will have access to certain information from that service, such as your profile information, in accordance with the authorization procedures determined by that service. Additionally, we may collect information about you from third-party sources that help us to learn more about you, like for example your profile picture or social media handles.
 
-### Use of Information
+## Legal Basis for Processing
+
+We process your personal information on the following legal bases:
+
+- **Performance of Contract**: Processing your information is necessary to provide the Services to you, such as creating an account, processing contributions, or handling expense reimbursements.
+- **Legitimate Interest**: We process your information for our legitimate business interests, such as improving our Services, understanding how our Services are used, preventing fraud, and marketing our Services.
+- **Consent**: In some cases, we process your information based on your consent, such as for certain types of marketing communications.
```

Comment:
Worth looking at but you have this basis covered in service provision and legitimate interest. Not sure this needs to be stated here as a result. Note that of course we have to gain explicit informed and free consent at the point that we register users to be able to send marketing and other non-service-provision based messaging.